src/shc.c

```diff
@@ -149,6 +150,9 @@ static int MMAP2_flag = 0;
 static const char BUSYBOXON_line[] =
 "#define BUSYBOXON	%d	/* Define as 1 to enable work with busybox */\n";
 static int BUSYBOXON_flag = 0;
+static const char PIPESCRIPT_line[] =
+"#define PIPESCRIPT	%d	/* Define as 1 to submit script as a pipe */\n";
+static int PIPESCRIPT_flag;
```

Comment:
Please initialize to 0

Reply:
OK, but static variables are initialized to 0 anyway

Reply:
I'll do, but if you feel it is necessary to modify something, you should have push access to my fork

Reply:
Hi, good job @fabio-brugnara, the PR fix the most important limitations of command line length limits. I'm chipping in to say that it's common practice for the PR raiser to fix things that the author reviewed, but I'm sure you know that as well. I know you're enjoying the fix yourself, but hope your contribution can serve the world as well. (I'm the DM of the shc Debian package, thanks)

Reply:
Thanks @suntong, actually my intention was to follow author's indication and I immediately  pushed a commit addressing the issue as requested, even if I felt it was not necessary. Maybe this is not the correct way to proceed? Is there something I overlooked?

Reply:
Thanks @fabio-brugnara. I saw "New changes since you last viewed". Hope @neurobin can be satisfied to take the PR. 

Hi @ashamedbit, would you take a look at this as well please?